samples/typescript/kafka-trigger/UsersTriggerMany/function.json

```diff
@@ -0,0 +1,15 @@
+{
+  "bindings": [
```

Comment:
this file should not be checked in correct?

Reply:
Yes. It is correct. for the other language implementation, we need to write function.json manually. They don't have a type. :)